integration-test/features/payments.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Create a spontaneous payment
+  Background:
+    Given GPS running
+
+  # Case OK
+  Scenario: An organization creates a spontaneous payment
+    Given the organization "777777" with an enrollment to service "service-1"
+    When the organization creates a spontaneous payment
+    Then the organization gets the created status code 201 
+    And the organization gets the created Payment Position
+    
+  # Case KO: 404 non-existent service
+	  Scenario: An organization tries to create a spontaneous payment on a service that doesn't exist
+	  Given the organization creates the creditor institution "777777"
+	  When the organization enrolls the creditor institution on the service "service-not-found" 
+	  Then the organization gets the created status code 404
+  
+  # Case KO: 400 bad request - non-existent property in service configuration
+	  Scenario: An organization tries to create a spontaneous payment with a not configured property request
+	  Given the organization "777777" with an enrollment to service "service-1"
+	  When the organization creates a spontaneous payment with a not configured property in the request
```

Comment:
change this 
`When the organization creates a spontaneous payment with a not configured property in the request`
in 
`When the organization creates a spontaneous payment without <PROP> property in the request`

Reply:
The test verifies that an error occurs when a property that is not configured on the service present in the db is passed.
Maybe I could change to:

When the organization tries to create a spontaneous payment with a "property-not-found" that is not expected in the service configuration

Reply:
or something like that `When the organization tries to create a spontaneous payment with a property "property-not-found" not configured to service`